.github/workflows/wheels.yml

```diff
@@ -0,0 +1,107 @@
+    steps:
+      - name: Checkout f3d-superbuild
+        uses: actions/checkout@v3
+        with:
+          submodules: true
+          path: 'f3d-superbuild'
+
+      # TODO: We are currently building wheels for master, it should be possible to select a tag
+      - name: Checkout f3d
+        uses: actions/checkout@v3
+        with:
+          repository: 'f3d-app/f3d'
+          ref: 'master'
+          fetch-depth: 0
+          path: 'f3d'
+
+      - name: Recover VTK version to build
+        working-directory: ${{github.workspace}}
+        shell: bash
+        run: |
+          echo VTK_COMMIT_SHA=`cat ./f3d/.github/actions/vtk_commit_sha` >> $GITHUB_ENV
+
+      - name: Setup Ninja Windows
+        if: runner.os == 'Windows'
+        shell: bash
+        run: |
+          choco install ninja
+          echo CMAKE_GENERATOR=Ninja >> $GITHUB_ENV
+
+      - name: Setup MSVC Windows
+        if: runner.os == 'Windows'
+        uses: TheMrMilchmann/setup-msvc-dev@v2
+        with:
+          arch: x64
+
+      - name: Escaping backslashes Windows
+        if: runner.os == 'Windows'
+        shell: bash
+        run: echo "ESCAPED_WS=${{github.workspace}}" | tr '\\' '/' >> $GITHUB_ENV
+
+      - name: Build wheels
+        uses: pypa/cibuildwheel@v2.13.1
+        with:
+          package-dir: ${{github.workspace}}/f3d
+          output-dir: ${{github.workspace}}/wheelhouse
+          config-file: ${{github.workspace}}/f3d/pyproject.toml
+        env:
+          CIBW_SKIP: '*-musl*'
+          # TODO: cross-compile ARM too
+          CIBW_ARCHS: native
+          CIBW_BUILD_VERBOSITY: 1
+          CIBW_ENVIRONMENT: >
+            CMAKE_PREFIX_PATH=../f3d-superbuild-build/install
+            SKBUILD_CMAKE_ARGS="
+            -DF3D_DEPENDENCIES_DIR=${{ env.ESCAPED_WS }}/f3d-superbuild-build/install/bin;
+            -DF3D_PLUGIN_BUILD_ALEMBIC=ON;
+            -DF3D_PLUGIN_BUILD_ASSIMP=ON;
+            -DF3D_PLUGIN_BUILD_DRACO=ON;
+            -DF3D_PLUGIN_BUILD_OCCT=ON;
+            -DF3D_PLUGIN_OCCT_COLORING_SUPPORT=ON;
+            -DF3D_MODULE_EXR=ON;
+            -DF3D_MODULE_EXTERNAL_RENDERING=ON"
+          CIBW_ENVIRONMENT_PASS_LINUX: >
+            CMAKE_PREFIX_PATH
+            SKBUILD_CMAKE_ARGS
+          CIBW_REPAIR_WHEEL_COMMAND: ''
+          CIBW_BEFORE_ALL: >
+            cmake -S f3d-superbuild -B f3d-superbuild-build
+            -DBUILD_SHARED_LIBS=OFF
+            -DCMAKE_BUILD_TYPE=Release
+            -DENABLE_f3d=OFF
+            -DENABLE_alembic=ON
+            -DENABLE_assimp=ON
+            -DENABLE_draco=ON
+            -DENABLE_occt=ON
+            -DENABLE_openexr=ON
+            -DENABLE_ospray=OFF
+            -DENABLE_pybind11=ON
+            -DENABLE_tbb=ON
+            -DENABLE_vtk=ON
+            -DOCCT_ENABLE_COLORING=ON
+            -Dvtk_SOURCE_SELECTION=git
+            -Dvtk_GIT_TAG=${{ env.VTK_COMMIT_SHA }} &&
+            cmake --build f3d-superbuild-build
```

Comment:
could that be done in an actual step instead of CIBW ?

Reply:
I don't think so, it's done in its own Docker container, and it's run for each cpu architecture